index.html

```diff
@@ -7,9 +7,9 @@
     <title>CodersCamp2020 | Super Farmer</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a7b2efdc17.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="styles/App.scss" />
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
```

Comment:
Why is Roboto gone? Was it moved somewhere else?

Reply:
I imported Roboto in 2 font weights. The import is placed above the import of our style file.